### DISCUSSION TITLE
Using YouTubes native captions not possible?

### DISCUSSION THREAD

**Discussion by brittonwalker** [vidstack/player#1367] [Q&A]
I've been trying to piece together the ability to use YT caption functionality within the player. I'm using the Plyr layout, but it seems like the only way to have captions is to upload captions yourself with the video to the player component, is this true?

**Comment by mihar-22** [comment]
Yes that's true as there's no captions API for us to load them through the embed. You can opt to just show the native embed controls if the YouTube captions are needed by setting `controls` on the player and removing the custom layouts.
